### PR TITLE
[REF] pre_commit_vauxoo: Merge vauxoo hooks into repo

### PR DESCRIPTION
**Pull request by antonag32**
Related: https://github.com/Vauxoo/pre-commit-vauxoo-hooks/issues/4

### CODE REVIEW

**Comment by antonag32** [comment]
@moylop260 Here is a possibility for merging hooks into pre_commit_vauxoo. I am still looking into other options as I don't like the idea of having to manually declare binaries for every new hook.

**Comment by antonag32** [comment]
Nevermind, seems like it is the proper way to do it. Used by the official hook repos of pre-commit. Please review @moylop260.

**Comment by moylop260** [comment]
LGTM!

👍

File: src/pre_commit_vauxoo/cfg/.pre-commit-config.yaml [hunk 1]
```diff
@@ -50,7 +50,13 @@ repos:
...
-    rev: v0.1.0
+  - repo: local
     hooks:
       - id: vx-check-deactivate
+        name: deactivate.jinja syntax check.
+        description: Validate deactivate.jinja files' syntax.
+        entry: vx-check-deactivate
+        args: []
+        language: python
+        types: [ jinja ]
```
Comment (moylop260) [inline src/pre_commit_vauxoo/cfg/.pre-commit-config.yaml:61]: I just noticed this extra spaces

I'm curious about this

O could be stripped?

Comment (antonag32) [reply src/pre_commit_vauxoo/cfg/.pre-commit-config.yaml]: They can be stripped, just did it.

**Pull request merged**